### PR TITLE
Add BlobStore metrics to grafana

### LINKED ISSUE #4393: Add blobstore metrics to dashboard
### Describe the feature

new blobstore txpool metrics:

https://github.com/paradigmxyz/reth/blob/2fc574f32999fbfd966ad137c9b3cfc190e48e48/crates/transaction-pool/src/metrics.rs#L42-L53

## TODO 
add to grafana dashboard, see existing txpool graphs:

https://github.com/paradigmxyz/reth/blob/main/etc/grafana/dashboards/overview.json

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by JosepBove**
Adds the metrics specified in  #4393

### CODE REVIEW

**Review by mattsse — APPROVED** [review]
lgtm

pending @shekhirin

File: etc/grafana/dashboards/overview.json [hunk 1]
```diff
@@ -5704,8 +5855,8 @@
   },
   "timepicker": {},
   "timezone": "",
-  "title": "reth",
+  "title": "Prometheus",
```
Comment (shekhirin) [inline etc/grafana/dashboards/overview.json:5858]:
```suggestion
  "title": "reth",
```

File: etc/grafana/dashboards/overview.json [hunk 2]
```diff
@@ -5704,8 +5855,8 @@
...
   "uid": "2k8BXz24x",
-  "version": 8,
+  "version": 4,
```
Comment (shekhirin) [inline etc/grafana/dashboards/overview.json:5860]:
```suggestion
  "version": 9,
```

**Review by shekhirin — CHANGES_REQUESTED** [review]
LGTM other than these 2 comments

**Comment by JosepBove** [comment]
Resolved

**Review by shekhirin — APPROVED** [review]

**Pull request merged**